src/SFA.DAS.EAS.Web/Views/EmployerManageApprentices/Details.cshtml

```diff
@@ -29,13 +29,84 @@
             <div class="panel panel-border-wide">
                 <h3 class="heading-small">Changes pending</h3>
                 <p>
-                The employer has made a change which needs to be approved by the training provider
+                    The employer has made a change which needs to be approved by the training provider
                 <p>
                     <a href="@Url.RouteUrl("ViewChanges", "EmployerManageApprentices")">View changes</a>
                 </p>
             </div>
         }
 
+        @if (Model.Data.HasPendingChangeOfProviderRequest
+            && Model.Data.PendingChangeOfProviderRequestWithParty.HasValue)
+        {
+            <div class="panel panel-border-wide alert-default">
+                @if (Model.Data.PendingChangeOfProviderRequestWithParty.Value == Party.Employer)
+                {
+                    <h3 class="heading-small">Changes for review</h3>
+                    <p>
+                        There are changes to this apprentice's details that you need to review.
+                    </p>
+                }
+                else
+                {
+                    <h3 class="heading-small">Changes pending</h3>
+                    <p>
+                        There are changes to this apprentice's details that are waiting for approval by the training provider.
+                    </p>
+
+                }
+                <p>
+                    <a id="changeOfPartyViewChangesLink" href="">View changes</a>
+                </p>
+            </div>
+        }
+
+
+        @if (Model.Data.HasApprovedChangeOfPartyRequest || Model.Data.IsContinuation)
+        {
+
+            <div class="panel panel-border-wide alert-default">
+
+
+                @if (Model.Data.HasApprovedChangeOfPartyRequest && !Model.Data.IsContinuation)
+                {
+                    <h3 class="heading-small">  View apprentice record </h3>
+                    <p>
+                        The apprentice moved providers during their apprenticeship.
+                    </p>
+                }
+                else if (Model.Data.HasApprovedChangeOfPartyRequest)
+                {
+                    <h3 class="heading-small">  View apprentice record </h3>
+                    <p>
+                        The apprentice moved providers during their apprenticeship.
```

Comment:
the text accroding to the content deck : The apprentice moved providers during their apprenticeship.

Reply:
wordings are same, hence no change